src/main/java/seedu/taassist/logic/commands/DeletecCommand.java

```diff
@@ -45,14 +52,20 @@ public CommandResult execute(Model model) throws CommandException {
                     model.getModuleClassList()));
         }
 
-        List<Student> students = model.getFilteredStudentList();
+        List<Student> students = model.getTaAssist().getStudentList();
```

Comment:
Perhaps it would be better if we can move `deleteModuleClass` logic to `Model` to improve abstraction. However, this is servicable for now.

Reply:
Mentioned in new issue #151.